tests/test_finder.py

```diff
@@ -692,6 +692,11 @@ def test_text_nomatch(self) -> None:
                 # TODO: deprecate OpenCV 3.X versions after time
                 if cv2.__version__.startswith("3.") and tdetect == "east":
                     continue
+                # TODO: Opencv version 4.9.0 fails here on docker CIs (C++ exception) with tesseract
+                if ocr == "tesseract":
+                    cv_version = [int(v) for v in cv2.__version__.split(".")]
+                    if cv_version[0] >= 4 and cv_version[1] >= 6:
```

Comment:
Dou mean 4.9.0 in the TODO or 4.6.0? How did you conclude any version after 4.6.0 and before 4.9.0 has the problem?

Reply:
any version above 4.6 the comment is wrong